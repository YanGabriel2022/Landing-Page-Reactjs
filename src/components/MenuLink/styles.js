import styled from "styled-components";
import { css } from "styled-components";
export const Container = styled.a`
  ${({ theme }) => css``}
`;
