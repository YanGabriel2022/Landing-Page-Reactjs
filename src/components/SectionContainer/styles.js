import styled, { css } from "styled-components";

export const Container = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    margin: 0 auto;
    width: 100%;
    padding: ${theme.spacings.large};
  `}
`;
