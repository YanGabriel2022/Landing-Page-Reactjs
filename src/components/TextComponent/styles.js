import styled from "styled-components";
import { css } from "styled-components";
export const Container = styled.div`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
  `}
`;
