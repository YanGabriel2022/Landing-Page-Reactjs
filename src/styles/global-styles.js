import { createGlobalStyle, css } from "styled-components";
export const GrobalStyles = createGlobalStyle`

@import url('https://fonts.googleapis.com/css2?family=Inter&family=Quicksand&family=Raleway&family=Roboto:ital,wght@1,300&family=Ubuntu:wght@700&display=swap');

* {
  margin: 0;
  padding: 0;
}
html{
  font-size: 62.5%;
}
body{
  font-size: 1.6rem;
  font-family:  ${({ theme }) => theme.font.family.default};
}
h1, h2, h3, h4, h5, h6{
  font-family: ${({ theme }) => theme.font.family.secondary}
}
`;
