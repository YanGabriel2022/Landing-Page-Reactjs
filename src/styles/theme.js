export const theme = {
  colors: {
    primaryColor: "#0A1128",
    secondaryBg: "#dc143c",
    white: "#FFFFFF",
  },
  font: {
    family: {
      default: "'Inter', sans-serif",
      secondary: "'Ubuntu', sans-serif",
    },
  },
};
