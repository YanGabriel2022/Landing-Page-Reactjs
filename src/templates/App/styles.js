import styled, { css } from "styled-components";
export const Wrapper = styled.div`
  /* ${({ background, theme }) => css`
    background: ${theme.colors.mainBg};
  `} */
`;
