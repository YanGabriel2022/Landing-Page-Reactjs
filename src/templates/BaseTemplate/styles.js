import styled from "styled-components";
import { css } from "styled-components";
export const Container = styled.div`
  ${({ theme }) => css``}
`;
