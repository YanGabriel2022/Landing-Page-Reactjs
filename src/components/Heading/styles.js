import styled, { css } from "styled-components";
import { theme } from "../../styles/theme";
export const Title = styled.h1`
  ${({ theme, colorDark }) => css`
    color: ${colorDark ? theme.colors.white : theme.colors.primaryColor};
  `}
`;
