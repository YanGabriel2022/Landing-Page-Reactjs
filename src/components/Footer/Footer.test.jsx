import { renderTheme } from "../../styles/renderTheme";
import { Footer } from ".";

describe("<Footer />", () => {
  it("should render", () => {
    const { container } = renderTheme(<Footer footerHtml="<h1>testeu</h1>" />);
    expect(container).toMatchSnapshot();
  });
});
