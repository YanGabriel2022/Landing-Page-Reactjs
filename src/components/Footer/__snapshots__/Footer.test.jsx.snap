// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
.c1 {
  max-width: 120rem;
  margin: 0 auto;
  width: 100%;
  padding: 3.2rem;
}

.c2 {
  font-size: 2.4rem;
}

.c0 {
  text-align: center;
}

.c0 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h1>
          testeu
        </h1>
      </div>
    </div>
  </div>
</div>
`;
